platform.md

```diff
@@ -949,7 +949,8 @@ The launcher:
 
 Given stack metadata containing `run-image.image` and a set of `run-image.mirrors`. The `<run-image>` for a given `<image>` shall be resolved as follows:
 - **If** any of `run-image.image` or `run-image.mirrors` has a registry matching that of `<image>`, this value will become the `<run-image>`
-- **If** none of `run-image.image` or `run-image.mirrors` has a registry matching that of `<image>`, `<run-image.image>` will become the `<run-image>`
+- **If** the `run-image.image` is accessible with read permissions, this value will become the `<run-image>`
+- The first value from the `run-image.mirrors` accessible with read permissions will become the `<run-image>`
```

Comment:
```suggestion
**If** any of `run-image.image` or `run-image.mirrors` is accessible with read permissions and has a registry matching that of `<image>`, this value will become the `<run-image>`
**If** none of `run-image.image` or `run-image.mirrors` has a registry matching that of `<image>`, the first value of `<run-image.image>` or `run-image.mirrors` that is accessible with read permissions will become the `<run-image>`
```

I can't make a code suggestion on line 951 as it didn't change, but I think we'd want to check read permissions there as well.